### DISCUSSION TITLE
补帧快进闪退

### DISCUSSION THREAD

**Discussion by playeriv65** [hooke007/mpv_PlayKit#446] [Q&A 问答]
补帧预设为MEMC_RIFE_NV.vpy ， 正常观看可以为4k视频补帧，CPU显卡内存显存剩余资源均充足，但是只要快进一大段就有概率闪退。如果先切回不补帧状态，再快进，再切回补帧状态，则可以正常继续观看。

**Comment by hooke007** [comment]
尝试禁用ext_proc

**Reply by playeriv65** [reply]
不知道为什么又不闪退了，试了半天也无法复现闪退，那就先用着吧。

**Reply by playeriv65** [reply]
ext_proc默认是禁用的，现在又复现了闪退，且每次闪退后显存占用都会提升400MB左右。怀疑闪退导致了显存泄露，后面播放视频的时候显存越来越少，继续闪退产生恶性循环。

**Reply by hooke007** [reply]
我说反了，尝试启用ext_proc

**Reply by hooke007** [reply]
闪退之后看看任务管理器里是不是存在mpv的进程残留
